src/OutputManager.php

```diff
@@ -81,20 +80,24 @@ private function setOutputDirectory()
      * Append a class to a file and save it
      * If no file is created the name of the class is the filename
      *
-     * @param PhpClass $class
+     * @param ZendClassGenerator $class
      */
-    private function saveClassToFile(PhpClass $class)
+    private function saveClassToFile($class)
     {
         if ($this->isValidClass($class)) {
-            $file = new PhpFile($class->getIdentifier());
+            $file = new FileGenerator();
+            $file->setFilename($class->getName() . '.php');
 
             $namespace = $this->config->get('namespaceName');
             if (!empty($namespace)) {
-                $file->addNamespace($namespace);
+                // This will _overwrite_ attached class namespace no _null_!
+                $file->setNamespace($this->config->get('namespaceName'));
```

Comment:
Do we need to do this add all since we [set the namespace for the individual classes](https://github.com/wsdl2phpgenerator/wsdl2phpgenerator/pull/255/files#diff-16a921c287a57770d0ab59c0fb771b5dR160)?